src/main/java/org/fcrepo/camel/processor/SparqlUpdateProcessor.java

```diff
@@ -49,38 +56,19 @@ public void process(final Exchange exchange) throws IOException {
         final Message in = exchange.getIn();
 
         final ByteArrayOutputStream serializedGraph = new ByteArrayOutputStream();
-        final String subject = ProcessorUtils.getSubjectUri(in);
-        final String namedGraph = in.getHeader(FcrepoHeaders.FCREPO_NAMED_GRAPH, String.class);
-        final Model model = createDefaultModel().read(in.getBody(InputStream.class), subject,
-                langFromMimeType(in.getHeader(Exchange.CONTENT_TYPE, String.class)));
+        final String namedGraph = in.getHeader(FCREPO_NAMED_GRAPH, "", String.class);
+        final Model model = createDefaultModel();
+        final String subject = getSubjectUri(in).orElseThrow(() -> new IOException("Could not extract Subject URI"));
+
+        read(model, in.getBody(InputStream.class),
```

Comment:
I thought `RDFDataMgr::loadModel` is used for loading RDF from a file or URI and not from an InputStream. i.e. from the Jena docs: `loadModel("data.ttl")`